schema/src/main/scala/org/constellation/schema/serialization/SchemaKryoRegistrar.scala

```diff
@@ -102,7 +101,6 @@ object SchemaKryoRegistrar
         (classOf[SnapshotProposalPayload], 200),
         (classOf[Signed[SnapshotProposal]], 201),
         (classOf[SnapshotProposal], 202),
-        (classOf[MajorityInfo], 203),
-        (classOf[HeightRange], 204)
+        (classOf[FilterData], 203)
```

Comment:
Not sure if we should reuse the registration id for a different class. If we are not putting this value in a snapshot it's probably ok though.

Reply:
Yep, it's only used in the API, so it should never get mixed.